.evergreen/config.yml

```diff
@@ -761,6 +769,12 @@ tasks:
     commands:
       - func: install dependencies
       - func: run atlas tests
+  - name: test-ldap-auth
```

Comment:
FWIW it may be very useful to ensure that these auth tests are all similarly named, it was a little tricky to find the "green box" in EVG for this task, but now I know it's ordered alphanumerically `test-ldap` comes after `test-latest-*`. Maybe we want to rename them `test-auth-ldap`, `test-auth-kerberos` etc so they are grouped together